File: js/taller.js/25.js
console.log("--- MENÚ ---");
console.log("1. Saludar");
console.log("2. Mostrar fecha (simulada)");
console.log("3. Salir");

let opcion = prompt("Seleccione una opción (1-3):");

if (opcion === "1") {
    console.log("¡Hola! Esperamos que tengas un excelente día.");
} else if (opcion === "2") {
    console.log("Fecha simulada: 18 de Mayo de 2026");
} else if (opcion === "3") {
    console.log("Saliendo del programa... ¡Adiós!");
} else {
    console.log("Opción no válida.");
}

File: py/taller.py/25.py
print("--- MENÚ ---")
print("1. Saludar")
print("2. Mostrar fecha (simulada)")
print("3. Salir")

opcion = input("Seleccione una opción (1-3): ")

if opcion == "1":
    print("¡Hola! Esperamos que tengas un excelente día.")
elif opcion == "2":
    print("Fecha simulada: 18 de Mayo de 2026")
elif opcion == "3":
    print("Saliendo del programa... ¡Adiós!")
else:
    print("Opción no válida.")